src/main/java/io/confluent/connect/jdbc/sink/BufferedRecords.java

```diff
@@ -112,13 +112,16 @@ public BufferedRecords(
       // Continue with current batch state
       records.add(record);
       if (records.size() >= config.batchSize) {
+        log.debug("Flushing buffered records after exceeding configured batch size.");
```

Comment:
How about including `records.size()` and maybe even `config.batchSize` values in the message? Maybe:
```suggestion
        log.debug(
            "Flushing buffered records after number of records {} exceeds configured batch size {}.", 
            records.size(), 
            config.batch.size
        );
```

Reply:
I decided to put the flushed size log into the `flush()` method itself so I can be sure to catch it in every code path within the flush itself. I will add the batch size